skbio/sequence/_sequence.py

```diff
@@ -491,7 +491,23 @@ def _string(self):
 
     def __init__(self, sequence, metadata=None,
                  positional_metadata=None):
-        if isinstance(sequence, Sequence):
+
+        if type(sequence) == np.ndarray or isinstance(sequence, np.ndarray):
```

Comment:
It's definitely redundant. I had thought I benchmarked it as faster but I just double checked and the difference seems negligible. Removed.